services/diff/local.go

```diff
@@ -100,6 +100,9 @@ func (l *local) Apply(ctx context.Context, er *diffapi.ApplyRequest, _ ...grpc.C
 	)
 
 	var opts []diff.ApplyOpt
+	if er.Decryptconfig != nil && len(er.Decryptconfig.Parametersjson) > 0 {
+		opts = append(opts, diff.WithDcParametersJSON(er.Decryptconfig.Parametersjson))
```

Comment:
is it possible to do `diff.WithDecryptConfig` or is there a serialization issue?

Reply:
Thanks. Can use the WithCryptoConfig introduced already.